### PR TITLE
Fix for [DLNA] Many log messages from Media Renderers trying to see old events #3104

### PR DESCRIPTION
**Pull request by BaronGreenback**
Null was not being checked for in RenewEventSubscription.

The function now returns an empty EventSubscriptionResponse if the subscription cannot be found.

### CODE REVIEW

**Comment by crobibero** [comment]
Can you redo this PR with just your change? Right now when I go to review it shows no files changed.

File: .gitignore [hunk 1]
```diff
@@ -277,3 +277,4 @@ BenchmarkDotNet.Artifacts
 web/
 web-src.*
 MediaBrowser.WebDashboard/jellyfin-web/
+/Common/Networking
```
Comment (dkanada) [inline .gitignore:280]: What's this entry?

Comment (BaronGreenback) [reply .gitignore:280]: Can ignore it - been having local GIT issues via visual studio.

This change should have been 1 file - commits were stuck locally with no way to sync.

Comment (BaronGreenback) [reply .gitignore:280]: My local GIT is screwed. Please ignore it.

**Comment by BaronGreenback** [comment]
Will delete this PR and start again. Git is showing no change from the original, but there is!!!

Will nuke the branch and redo.

**Pull request closed without merging**